README.md

```diff
@@ -1,42 +1,131 @@
-# sql-data-integrity-checker
+# sdic
```

Comment:
it's a pain to handle `sql-data-integrity-checker` and `sql_data_integrity_checker` depending on where it's called, and `sdic` is free on pypi so... ;)